canonical-web-and-design/juju.is
# Add docs

## Done
Integrated the documentation module and pointed it to the current juju docs topic on discourse.

## QA
- Go to the demo
- Click the docs link in the navigation
- Check that the pages load and are correct

Fixes https://github.com/canonical-web-and-design/juju-squad/issues/958
